docs/platform-setup.md

```diff
@@ -144,6 +144,31 @@ Unfortunately, `nexus-docker-cfg` and `jenkins-docker-cfg` can seem duplicated,
 
 - Kaniko expects to have a secret with the `Opaque` type and data to be stored in `data/config.json`.
 
+##### Allow Docker Secret to Be Replicated
+
+To enable secret replicator for `nexus-docker-cfg` , apply this patch on it.
+
+```bash
+  kubectl patch secret/nexus-docker-cfg -n ${NAMESPACE} --patch "\$(cat templates/nexus-docker-cfg-patch.yaml)"
+```
+
+You can create the new secret replica using this template file:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: nexus-docker-cfg
```

Comment:
I think you should provide another name for the replica, different form the source secret.
For instance: `nexus-docker-cfg-replica`.